### PR TITLE
Bug 1975784, added oc-compliance plug-in installation procedure

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1975784

Preview: https://deploy-preview-35189--osdocs.netlify.app/openshift-enterprise/latest/security/oc_compliance_plug_in/oc-compliance-plug-in-using?utm_source=github&utm_campaign=bot_dp#installing-oc-compliance_oc-compliance-plug-in-understanding

### CODE REVIEW

File: modules/oc-compliance-installing.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+. Extract the `oc-compliance` image to get the `oc-compliance` binary:
++
+[source,terminal]
+----
+$ mkdir oc-compliance
+----
++
+[source,terminal]
+----
+$ oc image extract registry-proxy.engineering.redhat.com/rh-osbs/openshift-oc-compliance@sha256:07d6c4ab7388584f13e7d207349d948327de7c235173f2c5dded56d7d41d1390 --path /:oc-compliance
```
Comment (mrogers950) [inline modules/oc-compliance-installing.adoc:19]: This registry location is not publicly accessible actually, so we need to specify the published repo: `registry.redhat.io/compliance/oc-compliance-rhel8`, Also, we shouldn't use a SHA tag (or else we would need to update the docs for every release).

The extract commands can be simplified to:
`$ podman run --rm --entrypoint /bin/cat registry.redhat.io/compliance/oc-compliance-rhel8 /usr/bin/oc-compliance > ~/.local/bin/oc-compliance`

File: modules/oc-compliance-installing.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[source,terminal]
+----
+W0611 20:35:46.486903   11354 manifest.go:440] Chose linux/amd64 manifest from the manifest list.
+----
+
+. Once the image is extracted, copy the binary from the `oc-compliance` directory:
++
+[source,terminal]
+----
+$ cp oc-compliance/bin/oc-compliance /<HOME>/.local/bin
```
Comment (mrogers950) [inline modules/oc-compliance-installing.adoc:33]: If you use my suggested command this part is not needed

**Comment by ahardin-rh** [comment]
@mrogers950 Thanks for the feedback! I made some edits.
@pdhamdhe PTAL

File: modules/oc-compliance-installing.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+= Installing the oc-compliance plug-in
+
+.Procedure
+
+. Extract the `oc-compliance` image to get the `oc-compliance` binary:
++
+[source,terminal]
+----
+$ mkdir oc-compliance
+----
```
Comment (pdhamdhe) [inline modules/oc-compliance-installing.adoc:15]: @ahardin-rh No need to create directory as well. We can remove this step. Thanks

**Comment by ahardin-rh** [comment]
@pdhamdhe Thanks! Updated.

**Pull request merged**